kotlin-code-style.md

```diff
@@ -36,48 +40,49 @@ class Person(val firstName: String, val lastName: String, var age: Int) {
 }
 ```
 
-#Extensions
+# Extensions
 
-I suggest we create one extension file for each Type we need :
+When adding extensions to external classes, make separate files for each type
+and name them _<Type>Extensions.kt_:
 
 ```
-- StringExtension.kt
-- BitmapExtension.kt
-- ContextExtension.kt
-...
+- StringExtensions.kt
+- BitmapExtensions.kt
+- ContextExtensions.kt
+- ...
 ```
 
-#Object Expressions and Declarations
+# Object Expressions and Declarations
 
-##Object expressions
+## Object expressions
 
-To avoid too long method, we should externalize object expressions as properties
-:
+To avoid writing methods that are too long, externalize object expressions as
+properties:
 
 ```kotlin
-class Person(val firstName: String, val lastName: String, var age: Int) {
+class Person {
 
     private val anAttribute = object : MyJavaInterface() {
-        //...
+        // ...
     }
 
-    public fun aMethod () {
-        button.setOnSomeClick( anAttribute )
+    fun aMethod() {
+        button.setOnSomeClick(anAttribute)
     }
 
 }
 ```
 
-#Delegated Properties
+# Delegated Properties
 
-##Standard Delegates
+## Standard Delegates
 
-We should use the standard delegates `lazy`as often as possible in order to have
-lazy loading properties :
+We should use the standard delegates `lazy` as often as possible in order to
+have lazy loading properties:
 
 ```kotlin
 val myValue : String by lazy {
-    someInit() //executed only at the first access
-    callToAFunctionReturningAStringValue()
+    someInit()
+    functionReturningAStringValue()
 }
 ```
```

Comment:
well in the end `lazy` delegates are not so much used, at least by me ahah. And it really depends of the context, so I think this part can be removed from the play book